tests/test_views.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+from django.core.urlresolvers import reverse
+from django.http import Http404
+
+from controlled_vocabularies import views
+from .factories import VocabularyFactory, TermFactory
+
+pytestmark = pytest.mark.django_db
+
+
+class TestAbout():
+
+    def test_status_ok(self, rf):
+        request = rf.get('/')
+        response = views.about(request)
+        assert response.status_code == 200
+
+    def test_template_used(self, client):
+        response = client.get(reverse('about'))
+        assert response.templates[0].name == 'vocabularies/about.html'
+
+
+class TestVocabularyList():
+
+    def test_status_ok(self, rf):
+        request = rf.get('/')
+        response = views.about(request)
+        assert response.status_code == 200
+
+    def test_template_used(self, client):
+        response = client.get(reverse('vocabulary_list'))
+        assert response.templates[0].name == 'vocabularies/vocabulary_list.html'
+
+    def test_context_var(self, client):
+        VocabularyFactory.create_batch(10)
+        response = client.get(reverse('vocabulary_list'))
+        assert response.context['vocabularies'].count() == 10
```

Comment:
Might add a note to this test that we should expect to find all of the vocabs in the DB.